package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@viaprotocol/web3-wallets",
+  "private": false,
+  "version": "1.0.1",
+  "author": "Aleksey Lisun<insuline.ru@gmail.com>",
```

```suggestion
  "author": "Via Protocol (https://via.exchange)",
```

More like collective authorship 😀